index.html

```diff
@@ -11,12 +11,12 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="styles.css" />
-    <title>Portfolio version 2</title>
+    <title>My Portfolio</title>
   </head>
   <body>
     <nav>
       <div class="nav">
```

Comment:
- [ ] Good job adding accessibility to your project 💯, but the [interaction](https://www.w3.org/WAI/test-evaluate/preliminary/#interaction) is not working as expected, please when you press the tab on your keyboard the focus should change between all elements, and in your project, the menu bar is not taking the focus, also your social media in the main section are not working as expected. Kindly fix it and request a new review. 🥇